index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <header>
+    <h1 id='title'> “Football Survey Form”</h1>
+    <p id='description'>Choose the best player(s) in Africa</p>
+  </header>
+  <form id="survey-form">
+    <label id="name-label">Name</label>
+    <input id="name" type="text" required placeholder="input your name">
+
+    <label id="email-label">Email</label>
+    <input id="email" type="email" required placeholder="input your email">
+
+    <label id="number-label">Age</label>
+    <input id="number" type="number" required max="70" min="18" placeholder="age">
+    <select id="dropdown">
+      <option>player1</option> 
+      <option>player2</option> 
+      <option>player3</option> 
+    </select>
+  <p> <h3>only one team selected</h3>
+    <input type="radio" id="" name="Team" value="PSG Team">
+    <label for="male">PSG</label><br>
+    <input type="radio" id="" name="Team" value="Barcelona">
+    <label for="female">Female</label><br>
+    <input type="radio" id="" name="Team" value="Real Madrid">
+    <label for="other">Other</label>
+  </p>
+
+  <p> <h3>One or more team selected</h3>
+    <input type="checkbox" id="" name="Team" value="PSG Team">
+    <label for="male">PSG</label><br>
+    <input type="checkbox" id="" name="Team" value="Barcelona">
+    <label for="female">Female</label><br>
+    <input type="checkbox" id="" name="Team" value="Real Madrid">
+    <label for="other">Other</label>
+  </p>
+  <p><textarea placeholder="additional comments"></textarea></p>
+  <p><input type="submit" id="submit" value="submit"></p>
+  </form>
+
+
```

Comment:
Please add inline comments to your HTML. comments make your code more structured and easier to read